index.html

```diff
@@ -30,7 +30,7 @@ <h1 id="temp"></h1>
         <h2 id="cityState"></h2>
         <h3> United States</h3>
         <div id="date"></div>
-        <div id="time"> 11:11</div>
+        <div id="time"></div>
```

Comment:
Currently, my html is very similar to yours, I know that with a refactor of the index.js, it would bring down the line numbers in here. I also know that there is a way to separate this out into other files. I have not yet implemented this either.